scripts/test_runner.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
```

Comment:
did you test adding `set -e` in travis.yml? if that works we may reduce the level of script calls.

Reply:
Good point. Let me try it out.